src/components/bulk-email-tool/bulk-email-form/BulkEmailRecipient.jsx

```diff
@@ -82,7 +82,6 @@ export default function BulkEmailRecipient(props) {
               <Form.Checkbox
                 key={cohort}
                 value={`cohort:${cohort}`}
-                disabled={selectedGroups.find((group) => group === (DEFAULT_GROUPS.AUDIT || DEFAULT_GROUPS.VERIFIED))}
```

Comment:
Is this the sneaky bugfix mentioned?

Reply:
yes - cohorts were being disabled when they weren't supposed to be because I forgot to delete this line in the copy paste. I just happen to notice it and it was a one-line fix.